src/templates/base-generated-config.yaml

```diff
@@ -46,7 +50,7 @@ jobs:
             fi
 
       - store_test_results:
-          path: << parameters.test-suite-path >>/result
```

Comment:
I don't think this line is compatible with next step using "export TEST_RESULT_REPORT_PATH=<< parameters.test-suite-path >>/result/example_result.message"

Reply:
I will match the second line like:
`export TEST_RESULT_REPORT_PATH=<< parameters.test-suite-path >>/result/<< parameters.bb-version >>_example_result.message
`

Reply:
Maybe its not needed now, I will revert it to << parameters.test-suite-path >>/result